src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php

```diff
@@ -28,7 +28,7 @@ class PostgresGrammar extends Grammar
      */
     public function compileTableExists()
     {
-        return 'select * from information_schema.tables where table_name = ?';
+        return 'select * from information_schema.tables where table_schema = ? and table_name = ?';
```

Comment:
@GrahamCampbell changing the sql considered breaking? It's a public method. Should I submit to master?